### DISCUSSION TITLE
<title>i want to solve this problem.

### DISCUSSION THREAD

**Discussion by whereismylife** [community/community#43471] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

I'm using the labkit in image j. 

But I couldn't get out of this problem.

please help me!

![화면 캡처 2023-01-06 180617](https://user-images.githubusercontent.com/106086190/211001350-d9a56046-4923-4ea8-81de-df7e79f50db2.png)

![image](https://user-images.githubusercontent.com/106086190/211001536-6d7f7399-5937-4104-a82a-70552f6d63c7.png)

**Comment by queenofcorgis** [comment, marked as answer]
Hi @whereismylife ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project, such as Labkit. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
